salt/ext/win_inet_pton.py

```diff
@@ -22,7 +22,7 @@ class sockaddr(ctypes.Structure):
     WSAStringToAddressA = ctypes.windll.ws2_32.WSAStringToAddressA
     WSAAddressToStringA = ctypes.windll.ws2_32.WSAAddressToStringA
 else:
-    def not_windows():
+    def not_windows(*args):
```

Comment:
WSAStringToAddressA and WSAAddressToStringA take arguments, so they are called with them below. I think it makes more sense for the body of this function to be executed and explain the problem than to throw a TypeError for being passed the wrong number of arguments. I found the issue here: https://lgtm.com/projects/g/saltstack/salt/snapshot/a3b59d268b5a7eafc74b7e1671dc64ce0159dea3/files/salt/ext/win_inet_pton.py#V44